packages/perspective-webpack-plugin/index.js

```diff
@@ -0,0 +1 @@
+module.exports = require("@jpmorganchase/perspective/webpack-plugin");
```

Comment:
When I saw this PR I was expecting to see `webpack-plugin` and its associated loaders moved into the `perspective-webpack-plugin` project. Is there any reason why that is not the case?